bsp/esp32_s2_kaluga_kit/include/bsp/display_backlight.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief BSP LCD backlight
+ *
+ * This file offers API for display backlight control
+ */
+
+#pragma once
+#include "esp_err.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Backlight functions are not implemented - Kaluga board doesn't provide backlight control
+   These functions are here to provide consistent API with other Board Support Packages */
+esp_err_t bsp_display_backlight_on(void);
```

Comment:
Can we add here  `esp_err_t bsp_display_brightness_set(int brightness_percent);` only for compatibility?

Reply:
Done, thanks!